### DISCUSSION TITLE
When the template is a table, how to use patchDocument to dynamically add a row of data

### DISCUSSION THREAD

**Discussion by yxengine** [dolanmiu/docx#2326] [Q&A] [answered]
template
![image](https://github.com/dolanmiu/docx/assets/129268404/6d395593-ec01-45e9-9015-783e7dc85946)

expect result:
![image](https://github.com/dolanmiu/docx/assets/129268404/45d83e5f-5439-424d-8dcc-dd25508682b7)

**Comment by JackVeromeev** [comment]
@dolanmiu , is that possible?

**Reply by dolanmiu** [reply, marked as answer]
This is not possible nope
patching is simply doing a find + replace, so this is not suited for your use case

**Comment by Qitao-Chen** [comment]
"docx-templates" library should be helpful
